code/src/OSFramework/OSStructure/ColumnProperties.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSStructure {
+    export enum ColumnProperties {
```

Comment:
@glundgren93 Just to start being consistent with other assets like OS UI, please order the options alphabetically